concrete/attributes/boolean/controller.php

```diff
@@ -27,6 +30,9 @@ public function searchForm($list)
         return $list;
     }
 
+    /**
+     * @since 5.7.0 (but not in 8.0.0 8.0.1 8.0.2 8.0.3 8.1.0 8.2.0)
```

Comment:
@mlocati IMO, this should be `@since 8.2.1`

Reply:
Why? He had it in previous versions ([example for 5.7.0](https://github.com/concrete5/concrete5/blob/5.7.0/web/concrete/attributes/boolean/controller.php#L80))...

Reply:
I see... but it looks like a bug. How about this style?

```
@since 8.0.0 Deleted.
@since 8.2.1 Re-implemented.
```